### PR TITLE
Refactor/general

### LINKED ISSUE #342: Unknown Requester using WebAuth Wallet
EarnProton.com should be a known requestor on the WebAuth wallet. 
image

### LINKED ISSUE #343: Wallet Links on Apple Store and Play Store should link to new WebAuth wallet
These buttons should link to new WebAuth wallet download links. 

Links here: https://webauth.com/login

After joining as a referral, there is a modal with a link to download a wallet. It is needed to update this link to webauth wallet

image

### LINKED ISSUE #344: User should be able to Filter by Status in the User Management Table in the Admin Section
Add a filter to this section to filter between status types "admin" or "referrer" 

image

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- Resolve #342 
- Resolve #343 
- Resolve #344 

### Steps to test

1. `make run`

#### CheckList

- [ ] Render of the proton logo in the Login Modal is appropiate
- [ ] Update download link for the wallet
- [ ] Accept WebAuth.com wallet authentication
- [ ] Filter in the `User Management` table by user role

### CODE REVIEW

File: webapp/src/context/state.context.js [hunk 1]
```diff
@@ -137,6 +137,9 @@ export const useSharedState = () => {
   const login = async () => {
     try {
       const { link, session } = await loginWallet(false)
+
+      if (!link || !session) return
```
Comment (leisterfrancisco) [inline webapp/src/context/state.context.js:141]: After the `@proton/web-sdk` update, when the modal is closed it didn't fire the catch, so I had to do a manual validation

File: webapp/src/routes/Admin/index.js [hunk 2]
```diff
@@ -721,6 +725,10 @@ const Admin = () => {
     })
   }, [])
 
+  useEffect(() => {
+    reloadUsers()
+  }, [filterRowsBy])
```
Comment (leisterfrancisco) [inline webapp/src/routes/Admin/index.js:730]: @tetogomez Is there another way to avoid this small useEffect code block?

**Review by xavier506 — APPROVED** [review]
LGTM

File: webapp/src/components/Accordion/index.js [hunk 3]
```diff
@@ -6,20 +6,42 @@ import AccordionSummary from '@material-ui/core/AccordionSummary'
...
   const classes = useStyles()
   const [expanded, setExpanded] = useState(false)
+  const [anchorEl, setAnchorEl] = useState(null)
 
   const handleAccordionExpand = () => {
     setExpanded(!expanded)
   }
 
+  const handleOpenMenu = event => {
+    event.stopPropagation()
```
Comment (tetogomez) [inline webapp/src/components/Accordion/index.js:37]: good one!

File: webapp/src/routes/Admin/index.js (hunk 2 above)
Comment (tetogomez) [reply webapp/src/routes/Admin/index.js:730]: it's work for me!

File: webapp/src/routes/Admin/index.js [hunk 4]
```diff
@@ -762,7 +770,12 @@ const Admin = () => {
           usePagination
         />
       </Accordion>
-      <Accordion title="User Management">
+      <Accordion
+        title="User Management"
+        filterValues={['All roles', 'Admin only', 'Referrers only']}
```
Comment (tetogomez) [inline webapp/src/routes/Admin/index.js:775]: filters values doesn't has i18n ?

**Review by tetogomez — CHANGES_REQUESTED** [review]
please check the comment related to i18n filter values

File: webapp/src/routes/Admin/index.js (hunk 2 above)
Comment (xavier506) [reply webapp/src/routes/Admin/index.js:730]: it works for me...

File: webapp/src/routes/Admin/index.js (hunk 4 above)
Comment (leisterfrancisco) [reply webapp/src/routes/Admin/index.js:775]: Seems that earnproton.com doesn't implement translation. In fact, some lines like [this](https://github.com/eoscostarica/proton-affiliate/blob/dev/webapp/src/routes/Admin/index.js#L732) don't also use `i18n`, for this reason, I decided to don't use 'i18n' either.

Please @tetogomez confirm me if It is right or if I would need to do that implementation.

Comment (xavier506) [reply webapp/src/routes/Admin/index.js:775]: i think it would be good to include `i18n` wherever possible

Comment (leisterfrancisco) [reply webapp/src/routes/Admin/index.js]: @tetogomez @xavier506 

I just update it, take a look again.

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**